src/createServer.js

```diff
@@ -1,11 +1,25 @@
+/* eslint-disable no-console */
```

Comment:
Consider removing the 'eslint-disable no-console' comment if you don't have any console statements in this file.